### PR TITLE
JETC-4372: Remove the part where pandokia setup imports pandokia

### PR DESCRIPTION
**Pull request by ariedel**
On the theory that the circular dependency is at the end of the setup.py file where pandokia setup tries to import pandokia, I tried removing it. Given that it was just loading pandokia to print out the name of this file... I don't think we need it.

This has now been tested by installing this branch on glitch; it worked!

### CODE REVIEW

**Comment by ariedel** [comment]
I'm actually going to hold off on merging this for now; the command that's still supposed to work (`pdk config`) doesn't.

**Comment by cdsontag** [comment]
Adric do we actually *use* that command?  Let's not get stuck trying to keep all of pdk alive, use it or not - we can't sustain that.

But assuming we do us it, another easy solution would be to, instead of deleting that code, just wrapping it in a try catch block.  If it catches *any* error during the import is prints a msg and moves on.

**Comment by ariedel** [comment]
pdk getenv is also not working, and neither is the pandokia website. Something else is broken.

**Comment by ariedel** [comment]
What appears to be broken is the fixes I put in for JETC-1051 (replace imp)

**Review by oiintam — APPROVED** [review]

**Comment by cdsontag** [comment]
With all the back and forth, to save Oi In time, can I suggest that you take over testing and deploy on glitch @ariedel ?  I hate to see things get suggested and Oi In spend any more time attempting and rolling back.

Or is this PR just fixing master so it works with TP16.1?

**Comment by ariedel** [comment]
This is about fixing master so it works with TP17, which I now believe I have done. It was a genuine circular dependency - to load common, you had to have already loaded cfg, which required (after a few steps) loading common...

Moving the new load_source-like function to importer.py and fixing the imports to other places does seem to have fixed it.

**Comment by ariedel** [comment]
To be clear: I have installed a new TP17 on glitch, and successfully `pip install .`'d pandokia on it. `pdk config` works, `pdk getenv` works.

**Comment by cdsontag** [comment]
All good.  Just doesn't seem right to ask Oi In to have to iterate any more.

**Comment by ariedel** [comment]
Yes, I'll take over installing TP17 on glitch once this is merged.

**Review by cdsontag — APPROVED** [review]

**Pull request merged**
